client/reader/site-stream/index.jsx

```diff
@@ -127,10 +128,6 @@ const SiteStream = React.createClass( {
 			title = this.translate( 'Loading Site' );
```

Comment:
I think the reason is that contrarily to other Reader subsections, the Discover controller didn't pass `setPageTitle` to `SiteStream`. Will try to think of a way to retain its previous behavior in a `DocumentHead` compatible way :-)